bitsandbytes/triton/matmul_perf_model.py

```diff
@@ -0,0 +1,208 @@
+import functools
```

Comment:
This file should have a reference comment to the triton upstream it was copies from, as well as suitable license attribution.

```suggestion
# Adapted from https://github.com/triton-lang/kernels/blob/eeeebdd8be7d13629de22d600621e6234057eed3/kernels/matmul_perf_model.py
# https://github.com/triton-lang/kernels is licensed under the MIT License.

import functools
```
should probably be enough.

Reply:
Done!